public/js/listeners.js

```diff
@@ -52,7 +52,11 @@ function startListeners() {
     });
     window.addEventListener('mousemove', onMouseMove);
 
-
+    window.addEventListener('copy', (event) => {
```

Comment:
By doing this, the listeners that do the regular copy/paste of circuits `document.addEventListener('copy', function(e) ...` stopped working.

Reply:
Will Fix this soon @satu0king Sir.

Reply:
@satu0king can you please suggest what can be done here ?

Reply:
@yash15601 I am actually not sure myself. 

The issue is this 
1. The canvas needs to have a set of listeners to do copy/paste in canvas
1. The property menu text boxes should work as well and the canvas listeners shouldn't work then
1. Delete key on mac is `backspace` and that needs to be supported, unfortunately in the textbox, backspace will delete of the element.
1. Once a textbox is selected, and the canvas is clicked, the focus needs to change back to the canvas.

Reply:
@satu0king   4 is working properly will check other

Reply:
> @yash15601 I am actually not sure myself.
> 
> The issue is this
> 
> 1. The canvas needs to have a set of listeners to do copy/paste in canvas
> 2. The property menu text boxes should work as well and the canvas listeners shouldn't work then
> 3. Delete key on mac is `backspace` and that needs to be supported, unfortunately in the textbox, backspace will delete of the element.
> 4. Once a textbox is selected, and the canvas is clicked, the focus needs to change back to the canvas.

Will fix these issues soon @satu0king Sir.